src/transformer.cpp

```diff
@@ -53,20 +57,9 @@ void Transformer::send(const std::string& id, TimeValue tv)
     data_channel_->publish(data_exchange_, id, DataChunk(tv).SerializeAsString());
 }
 
-void Transformer::config(const json& config)
+void Transformer::configure(const json& config)
```

Comment:
Should be `on_transformer_config` according to the current naming.

Reply:
Ok, that's wrong. Its really not the callback for the subclass, but a config + subscribe intermediate. I suggest to split it to a separate `subscribe_metrics` and call `on_transformer_config` + `subscribe_metrics` at the two places where now `configure` is called.